dev/tools/controllerbuilder/pkg/codegen/common.go

```diff
@@ -39,7 +39,8 @@ var protoMessagesNotMappedToGoStruct = map[string]string{
 
 var Acronyms = []string{
 	"ID", "HTML", "URL", "URI", "HTTP", "HTTPS", "SSH",
-	"IP", "GB", "FS", "PD", "KMS", "GCE", "VTPM",
+	"IP", "GB", "FS", "PD", "KMS", "GCE", "VTPM", "SSL",
+	"CA", "CPU", "SQL", "PSC", "DNS",
```

Comment:
Others looks good but I'm not sure about CA and PSC.

Reply:
Context about PSC and CA:
* PSC: https://cloud.google.com/vpc/docs/private-service-connect, https://medium.com/google-cloud/gcp-private-service-connect-psc-service-publication-62eaf1d58651
* CA: https://cloud.google.com/security/products/certificate-authority-service, https://medium.com/google-cloud/gcp-ca-service-how-to-get-started-6eba056c9e12